tests/unit/systems/dag/test_executors.py

```diff
@@ -152,7 +153,44 @@ def test_triton_executor_model(tmpdir):
     filtering = ["candidate_ids"] >> FilterCandidates(filter_out=["movie_ids"])
 
     ensemble = Ensemble(filtering, request_schema)
```

Comment:
This would probably be clearer if it either explicitly specified `runtime=TritonEnsembleRuntime()` (even though that's the default) or had a comment that explained what the default is (since it isn't explicitly in the code.)